<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ActionZoneChange</class>
 <widget class="QWidget" name="ActionZoneChange">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="lblZone">
       <property name="text">
        <string>Zone:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="zone">
       <property name="maximum">
        <number>2147483647</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblDynamicMap">
       <property name="text">
        <string>Dynamic Map:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="dynamicMap">
       <property name="maximum">
        <number>2147483647</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lblDestination">
       <property name="text">
        <string>Destination:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="ObjectPosition" name="destination" native="true"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ObjectPosition</class>
   <extends>QWidget</extends>
   <header>ObjectPositionUI.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
